org.kde.kbibtex.json

```diff
@@ -0,0 +1,80 @@
+{
+    "id": "org.kde.kbibtex",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15-21.08",
+    "sdk": "org.kde.Sdk",
+    "command": "kbibtex",
+    "rename-icon": "kbibtex",
+    "finish-args": [
+        "--share=ipc",
+        "--share=network",
+        "--socket=cups",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--filesystem=home"
```

Comment:
Is this really needed? Can you provide a short justification in a README?

Reply:
It is less than https://github.com/flathub/org.kde.kwrite/blob/master/org.kde.kwrite.json#L14 It is essentially a specialized file editor. It needs to read/save data somewhere.

Reply:
👍🏻

Reply:
Yes, it can save even without this explicit permission. Removed it.